Add moderator to Denver event

kimdhendrick opened the pull request.

rradczewski commented:
Hi @kimdhendrick, the correct field would be `moderators`, and it takes an array of values. Good occasion to disallow extra properties on the jsons 😥

rradczewski commented on _data/events/Denver.json:
```diff
@@ -8,6 +8,7 @@
     },
-    "utcOffset": -6
+    "utcOffset": -6,
+    "moderator": "@kimdhendrick"
```
See json schema, should be array of strings called `moderators`

kimdhendrick commented:
Hi,
I fixed the Denver.json file and also added `"additionalProperties": false` to prevent this in the future. I see the conflict that you already added that as well, but I left my changes as I also added a test for that change. Let me know if you want me to include those changes or just submit with only my Denver.json fix.

Thanks!
Kim

rradczewski commented:
Hi @kimdhendrick, thanks for your proposal! I will leave you a review in a second.

Gist is: Please submit separate PR's for these issues. The test code needs some changes in my opinion and I don't want to keep your change to the event from showing up on the map asap.

rradczewski commented on events/event_schema.json:
```diff
@@ -57,5 +57,6 @@
   },
-  "required": ["title", "location", "url"]
+  "required": ["title", "location", "url"],
+  "additionalProperties": false
```
`additionalProperties: false` needs to show up on every level of the schema as implemented already on [master](https://github.com/coderetreat/coderetreat.github.io/blob/master/events/event_schema.json). I think we can stick to the version on `master`, but I'd love to have you submit another PR for the test below.

rradczewski commented on test/validate_events.test.js:
```diff
@@ -59,7 +59,7 @@ describe('Events as given in /_data/events', () => {
 describe('Invalid events given in /test/invalid_events', () => {
   describe('that require at least 1 character: ', () => {
```
the description does not match the test case. I'd propose you delete the second level `describe`. So the reason the tests are invalid is reflected in the name of the json file.

rradczewski commented on test/validate_events.test.js:
```diff
@@ -59,7 +59,7 @@ describe('Events as given in /_data/events', () => {
 describe('Invalid events given in /test/invalid_events', () => {
   describe('that require at least 1 character: ', () => {
     const eventsWithEmptyFields = glob.sync(path.resolve(__dirname, './invalid_events/')
```
```
-eventsWithEmptyFields
+invalidEvents
```

kimdhendrick commented:
Ok, here's just the moderators change. 

Next PR will include tests for `additionalAttributes` changes in schema.

kimdhendrick commented:
Trying to separate these two changes into separate PRs!
